dashboard/config/routes.rb

```diff
@@ -53,12 +53,23 @@ module OPS
 
   get 'docs/*docs_route', to: 'docs_proxy#get'
 
-  resources :sections, only: [:index, :show, :update] do
+  # User-facing section routes
+  resources :sections, only: [:show, :update] do
     member do
       post 'log_in'
     end
   end
-  get '/dashboardapi/sections/', to: 'sections#index'
+  # Section API routes (JSON only)
+  namespace :api do
+    namespace :v1 do
+      resources :sections, only: [:show, :index]
+    end
+  end
+  # Temporarily expose Section API routes on dashboardapi, making them
+  # available to pegasus as well.
+  scope 'dashboardapi', module: 'api/v1' do
+    resources :sections, only: [:show, :index]
```

Comment:
Note: In a near-future PR I have a cleanup that makes the sections resource a concern, so we stop duplicating this line between the /api/v1 namespace and the /dashboardapi scope.